release-manager/files/README.md

```diff
@@ -60,17 +60,31 @@ repositories:
 
 This type designates ODS components designed for _code development_. Such repositories are based on quickstarters whose names start with `be-`, `ds-`, or `fe-`, for _backend_, _data science_, and _frontend_, respectively. This is the default type.
 
+If you use this type ODS expects to find JUnit XML test results. If you do not have any test results the pipeline will fail. If you are deploying something where JUnit XML test results are not available consider using <<Repository Type: ods-service>>.
```

Comment:
would it be feasible to add in which folder / filename?

Reply:
It is the test folder but it can be configured. We could indicate the default value and point to the documentation where it is mention how to change it